libs/core/dtos/register.dto.ts

```diff
@@ -0,0 +1,15 @@
+/** Register DTO. */
+export type RegisterDto = {
```

Comment:
```suggestion
export type RegistrationDto = {
```
Models must use nouns